### PR TITLE
Copy how cypress is run on the server

### PR DESCRIPTION
**Pull request by artonge**
This make e2e experience nicer and the code cleaner
This also takes advantages of sessions, so tests should run faster. Confirmed, tests ran in 3:48 min, where average for this repo was 6:04 min

### CODE REVIEW

File: cypress/dockerNode.ts [hunk 1]
```diff
@@ -0,0 +1,221 @@
...
+/**
+ * Configure Nextcloud
+ */
+export const configureNextcloud = async function() {
+	console.log('\nConfiguring nextcloud...')
+	const container = docker.getContainer(CONTAINER_NAME)
+	await runExec(container, ['php', 'occ', '--version'], true)
+
+	// Clone the viewer app
+	await runExec(container, ['git', 'clone', '--depth', '1', 'https://github.com/nextcloud/viewer.git', '/var/www/html/apps/viewer'], true)
```
Comment (skjnldsv) [inline cypress/dockerNode.ts:121]: Careful about the branch

Comment (artonge) [reply cypress/dockerNode.ts]: Changed to use the server's branch

File: cypress/e2e/albums.cy.js [hunk 2]
```diff
@@ -19,116 +19,126 @@
...
+	it('Download multiple files from an album', () => {
+		selectMedia([1, 2])
+		downloadSelection()
+		unselectMedia([1, 2])
+	})
+
+	it('Download all files from an album', () => {
+		selectMedia([1, 2])
+		downloadSelection()
+		unselectMedia([1, 2])
```
Comment (skjnldsv) [inline cypress/e2e/albums.cy.js:110]: Any reason not to make those cypress commands?

Comment (artonge) [reply cypress/e2e/albums.cy.js:110]: Mostly typings. Is there a reason to make them cypress commands ? I thought commands were the way to go, but it does not add anything in my experience.

**Comment by szaimen** [comment]
Sorry but I could onky give a 🐘 on this

File: tsconfig.json [hunk 3]
```diff
@@ -0,0 +1,22 @@
...
+		"target": "ESNext",
+		"module": "esnext",
+		"declaration": true,
+		"strict": true,
+		"noImplicitAny": false,
+		"resolveJsonModule": true
+	},
+	"ts-node": {
+		// these options are overrides used only by ts-node
+		// same as our --compilerOptions flag and our TS_NODE_COMPILER_OPTIONS environment variable
```
Comment (marcelklehr) [inline tsconfig.json:17]: Comments in JSON?

**Review by marcelklehr — APPROVED** [review]
Tests are passing :tada: :elephant:

File: tsconfig.json (hunk 3 above)
Comment (artonge) [reply tsconfig.json:17]: Looks like yeah. Don't seems to cause any issue :see_no_evil:

**Pull request merged**